### ISSUE TITLE
GitHub Actions fail on MR tests

### ISSUE THREAD

**Issue by KrisThielemans** [SyneRBI/SIRF#1267]
https://github.com/SyneRBI/SIRF/actions/runs/9556091923/job/26340671249#step:9:1053
```
E               ValueError: cannot fill AcquisitionData of size (np.int32(512), np.int32(8), np.int32(256)) with data of size (1, 1, 8, 131072)
```
@evgueni-ovtchinnikov this is rather strange, as the recent commit didn't change anything in Gadgetron.

**Comment by KrisThielemans** [comment]
@evgueni-ovtchinnikov this prevents images being build, so needs urgent attention. sorry

**Comment by evgueni-ovtchinnikov** [comment]
the failure happens when testing `test_imagedata_constructor.py`, but SIRF master does not have this script!

apparently, GitHub Actions do some other branch - how can we find which one?

**Comment by KrisThielemans** [comment]
we have https://github.com/SyneRBI/SIRF/blob/master/.github/workflows/build-test.yml
and https://github.com/SyneRBI/SIRF-SuperBuild/blob/master/version_config.cmake of course.

**Comment by evgueni-ovtchinnikov** [comment]
> the failure happens when testing test_imagedata_constructor.py, but SIRF master does not have this script!

found it (scrollbar does not show on VM unless you move the cursor to the window edge), sorry for the false alarm!

on my VM runs ok, so have to debug via GitHub Actions

**Comment by evgueni-ovtchinnikov** [comment]
looks fixed

likely reason was the change of numpy version, details tomorrow

**Comment by KrisThielemans** [comment]
ok. Indeed, we're getting numpy 2.0 now. It'd make sense that it's an external change causing this problem now.

**Comment by evgueni-ovtchinnikov** [comment]
fixed by 8e8d95bfb6423d6a6a794fafe88bca8cda8f4138
